<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bankwindow</class>
 <widget class="QWidget" name="bankwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Pääkäyttöliittymä</string>
  </property>
  <widget class="QPushButton" name="saldoButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Näytä saldo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tapahtumaButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>420</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selaa tilitapahtumia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nostoButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>570</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nosta rahaa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kirjauduUlosButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>570</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOmistaja">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>300</y>
     <width>531</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Omistajan nimi</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="labelLocked">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="talletusButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>700</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Talletus</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="timer">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>940</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
  </widget>
  <zorder>labelLocked</zorder>
  <zorder>talletusButton</zorder>
  <zorder>saldoButton</zorder>
  <zorder>tapahtumaButton</zorder>
  <zorder>nostoButton</zorder>
  <zorder>kirjauduUlosButton</zorder>
  <zorder>labelOmistaja</zorder>
  <zorder>timer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
